packages/web/src/features/translation/TranslateWord.tsx

```diff
@@ -137,12 +137,19 @@ const TranslateWord = forwardRef<TranslateWordRef, TranslateWordProps>(
               dir={targetLanguage?.textDirection ?? TextDirection.LTR}
             >
               <div className="group-focus-within/input-row:block hidden">
-                {status === 'saved' && (
+                {glossValue && status !== 'approved' && (
```

Comment:
Can we remove this? Right now if you have an empty gloss, and you type something, the button doesn't appear until you move off and come back
```suggestion
                {status !== 'approved' && (
```

Reply:
Then the approval button shows up for a completely empty input, which can't be approved.

Reply:
I added a `currentInput` state variable to help here. It seems to work well, but there might be edge cases I didn't catch.

Reply:
Can you try it out before we merge?